src/components/Form.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from "react";
+import { getUsers, postComment } from "../api";
+
+export default function Form(props) {
+  const [userList, setUserList] = useState("");
+
+  useEffect(() => {
+    getUsers().then((data) => {
+      setUserList(data);
+    });
+  }, []);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (e.target[1].value === "") {
+      alert("Please put something in the 'Your text here:' box.");
+    } else if (
+      userList
+        .map((user) => {
+          return user.username;
+        })
+        .includes(e.target[0].value)
+    ) {
+      postComment(props.review_id, e.target[0].value, e.target[1].value).then(
+        () => {
+          props.setSeed();
+        }
+      );
+    } else {
+      alert(
+        "Hmm... that username does not match our files. Please try another one."
+      );
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
```

Comment:
Sometimes the comments aren’t showing until you refresh (if you leave a comment by a different user) this may be resolved when adding the functionality to be logged in as only one user though. 

We would also not want to be able to click the post button more than once to prevent multiple posts of the same comment.  It would also be an idea to disable the form whilst the post request is submitting to prevent duplicate posts on slow connections.